File: ChuyenDeWeb-Pro-1/src/main/java/com/example/mypkg/entity/Product.java
package com.example.mypkg.entity;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.Table;

import com.example.mypkg.until.ProductType;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Entity
@AllArgsConstructor
@NoArgsConstructor
@Getter
@Setter
@Table(name = "product")
public class Product {
	@Id
	@GeneratedValue(strategy = GenerationType.IDENTITY)
	private Long id;
	private String code;
	private String productName;
	private Float price;
	private String img;
	private boolean isDelete;
	private ProductType productType;
}

File: app-news/src/model/product.ts
export class Product {
  id!: number;
  code!: string;
  productName!: string;
  price!: number;
  img!: string;
  quantity!: number;

  constructor(
    id: number,
    code: string,
    productName: string,
    price: number,
    img: string,
    quantity: number
  ) {
    this.id = id;
    this.code = code;
    this.productName = productName;
    this.price = price;
    this.img = img;
    this.quantity = quantity;
  }
}
